### PR TITLE
genymotion-cloud-saas-stop 0.0.1

### PR DESCRIPTION
**Pull request by thomascarpentier**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/genymotion-cloud-saas-stop/0.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+- ubuntu-16.04
+project_type_tags:
+- android
+- xamarin
+- react-native
+- cordova
+- ionic
+- flutter
+type_tags:
+- utility
```
Comment (koral--) [inline steps/genymotion-cloud-saas-stop/0.0.1/step.yml]: It should be be probably `test`

File: steps/genymotion-cloud-saas-stop/0.0.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+toolkit:
+  go:
+    package_name: github.com/genymobile/bitrise-step-genymotion-cloud-saas-stop
+is_requires_admin_user: true
+is_always_run: false
```
Comment (koral--) [inline steps/genymotion-cloud-saas-stop/0.0.1/step.yml]: Should be probably true (users want to stop remote work even if something before failed).

File: steps/genymotion-cloud-saas-stop/0.0.1/step.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+is_skippable: false
+run_if: ""
+inputs:
+- instance_uuid: ""
```
Comment (koral--) [inline steps/genymotion-cloud-saas-stop/0.0.1/step.yml]: It should default to `GMCLOUD_SAAS_INSTANCE_UUID` set by start step.

File: steps/genymotion-cloud-saas-stop/0.0.1/step.yml (hunk 1 above)
Comment (godrei) [reply steps/genymotion-cloud-saas-stop/0.0.1/step.yml]: see: https://github.com/bitrise-io/bitrise-steplib/pull/2344#discussion_r363683035

File: steps/genymotion-cloud-saas-stop/0.0.1/step.yml (hunk 2 above)
Comment (godrei) [reply steps/genymotion-cloud-saas-stop/0.0.1/step.yml]: agree

File: steps/genymotion-cloud-saas-stop/0.0.1/step.yml (hunk 3 above)
Comment (godrei) [reply steps/genymotion-cloud-saas-stop/0.0.1/step.yml]: agree

File: steps/genymotion-cloud-saas-stop/0.0.1/step.yml (hunk 1 above)
Comment (koral--) [reply steps/genymotion-cloud-saas-stop/0.0.1/step.yml]: see: https://github.com/bitrise-io/bitrise-steplib/pull/1101#issuecomment-314151310

**Pull request closed without merging**